src/nuclio/functions/version/version-triggers/version-triggers.component.js

```diff
@@ -82,6 +87,25 @@
         // Public methods
         //
 
+        /**
+         * Checks if classes should be disabled
+         */
+        function checkClassUniqueness() {
+            lodash.forEach(uniqueClasses, function (classId) {
+                var classData = lodash.find(ctrl.classList, ['id', classId]);
+                var isClassUsed = lodash.some(ctrl.triggers, function (trigger) {
```

Comment:
Please rename to `classIsUsed`.
For variables we should use this form.
For functions returning `boolean` we should use the `isSomething` form.